src/modules/solution-explorer/solution-explorer-panel/solution-explorer-panel.ts

```diff
@@ -141,6 +141,11 @@ export class SolutionExplorerPanel {
 
   public async openRemoteSolution(): Promise<void> {
     try {
+      const lastCharacterIsASlash: boolean = this.uriOfRemoteSolution.endsWith('/');
+      if (lastCharacterIsASlash) {
+        this.uriOfRemoteSolution = this.uriOfRemoteSolution.substring(0, this.uriOfRemoteSolution.length - 1);
```

Comment:
How about 
```suggestion
        this.uriOfRemoteSolution = this.uriOfRemoteSolution.slice(0, -1);
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/slice